packages/core/helper-plugin/src/utils/getDisplayName/index.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * Retrivies display name based on user data and with the formatMessage function to show the name if required
+ * @typedef AdminUserNamesAttributes
+ * @property {string} firstname
+ * @property {string} lastname
+ * @property {string} username
+ * @property {string} email
+ *
+ * @type {(user: AdminUserNamesAttributes, formatMessage: import('react-intl').formatMessage) => string}
+ */
+const getDisplayName = ({ firstname, lastname, username, email }, formatMessage) => {
+    if (username) {
+      return username;
+    }
+  
+    if (firstname) {
```

Comment:
I think firstname is a mandatory field for users, so we don't have to check for it. I don't know if SSO can be an exception? cc @Convly ?

Reply:
With SSO you can have either firstname + lastname or username (if I recall correctly)

Reply:
On second thought, this is not the first time I get asked this, maybe we could add a comment in this utils to add this context?
What do you think?